BUILDING.md

```diff
@@ -33,6 +33,10 @@ Support is divided into three tiers:
 
 ### Supported platforms
 
+The community does not build or test against end of life distributions (EoL).
+Thus we do not recommend that you use Node on end of life or unsupported platforms
+in production.
```

Comment:
Seems like perhaps this ought to also be stated in the README or somewhere else that someone downloading a binary might see it.  Ref: https://github.com/nodejs/node/issues/14182